client/packages/programs/src/JsonForms/common/components/Array/Notes.tsx

```diff
@@ -62,7 +62,7 @@ const NotesComponent = (props: ArrayControlCustomProps) => {
   const { enabled, data, config } = props;
   const { localisedDateTime } = useFormatDateTime();
 
-  const options = props.uischema.options;
+  const options = NotesOptions.parse(props.uischema.options);
```

Comment:
Nit: there is our `useZodOptionsValidation` hook which also returns some validation errors that are a bit nicer formatted and which can be shown in the UI (targeting developer)

Reply:
ooh.. with a `safeParse` that's much nicer 🙏

Reply:
Ah, right. Trying to remember why we weren't already using that hook in here -- I think it's because the options object is passed intact to the child `Note` components, and the zod validation is already done there. So didn't want to double up?

Anyway, makes sense to keep the same pattern regardless, so I'll make a small tweak in my next PR and note it. Thanks.